jcl/src/java.base/share/classes/com/ibm/tools/attach/target/CommonDirectory.java

```diff
@@ -111,7 +111,7 @@ private static void initialize() {
 	static void prepareCommonDirectory() throws IOException {
 		File cd = getCommonDirFileObject();
 		if (LOGGING_DISABLED != loggingStatus) {
-			IPC.logMessage("createDirectoryAndSemaphore ", cd.getAbsolutePath()); //$NON-NLS-1$
+			IPC.logMessage("getCommonDirFileObject() returns ", cd.getAbsolutePath()); //$NON-NLS-1$
```

Comment:
I don't think calling `cd.getAbsolutePath()` is free, perhaps it should be cached

Reply:
Removing this `IPC.logMessage` since each code path below now has the info of `cd.getAbsolutePath()`.